Description: Remove call to AM_GNU_GETTEXT from configure.ac
 Gettext conflicts with intltool when running autoreconf
 per https://bugs.launchpad.net/intltool/+bug/654918
Author: Felix Lechner <felix.lechner@gmail.com>
Forwarded: no
Last-Update: 2015-05-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -229,7 +229,6 @@
 if test "$mingw_found" != "yes" ; then
 	dnl gettext macro does not work properly under mingw. And we want to use the one provided by GTK.
 	CPPFLAGS_save=$CPPFLAGS
-	AM_GNU_GETTEXT([external])
 	AC_SUBST(INTLLIBS)
 	CPPFLAGS=$CPPFLAGS_save
 	LIBS="$LIBS $LIBINTL"
